tcbuilder/backend/combine.py

```diff
@@ -178,11 +184,15 @@ def combine_single_tezi_image(bundle_dir, files_to_add, output_dir, tezi_props):
     return update_tezi_files(output_dir, tezi_props, files_to_add)
 
 
-def check_combine_files(bundle_dir):
+def check_combine_files(bundle_dir, is_for_raw_image=False):
 
     files_to_add = []
     if bundle_dir is not None:
-        files_to_add = DOCKER_FILES_TO_ADD
+        if not is_for_raw_image:
+            files_to_add = DOCKER_FILES_TO_ADD
+        else:
+            files_to_add = DOCKER_FILES_TO_ADD_TO_RAW
+
```

Comment:
Related to a previous question: what's the relationship between the image being "raw" and the compression being active or not? Is this some kind of optimization?

Reply:
I guess this question has already been addressed in the previous conversation, right?

Reply:
Yes!